index.bs

```diff
@@ -323,15 +323,23 @@ they are also defined by how content uses those features.
 <h3 id="leave-the-web-better">Leave the web better than you found it</h3>
 
 As you add new capabilities to the web platform,
-do so in a way that improves the overall platform,
-for example its security, privacy or accessibility characteristics.
+do so in a way that improves the overall platform.
 
 The existence of a defect in one part of the platform
 must not be used to excuse an addition or extension to the defect,
 which would further decrease the overall platform quality.
-Where possible, build new web capabilities that improve the overall platform quality
-by mitigating existing defects.
-Do not degrade existing capabilities without good reason.
+
+Consistency is not a good reason to propagate problems.
+Where possible, build new web capabilities that improve the overall platform
+quality by mitigating existing defects.
```

Comment:
These are the same as the pre-change lines, so make sure they're also wrapped the same:

```suggestion
Where possible, build new web capabilities that improve the overall platform quality
by mitigating existing defects.
```